lib/shoulda/matchers/active_model.rb

```diff
@@ -1,7 +1,8 @@
 require 'shoulda/matchers/active_model/helpers'
 require 'shoulda/matchers/active_model/validation_matcher'
-require 'shoulda/matchers/active_model/validation_message_finder'
-require 'shoulda/matchers/active_model/exception_message_finder'
+require 'shoulda/matchers/active_model/validator'
+require 'shoulda/matchers/active_model/strict_validator'
+require 'shoulda/matchers/active_model/validator_with_captured_range_error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.